cli/src/cmd/forge/tree.rs

```diff
@@ -29,7 +28,7 @@ impl Cmd for TreeArgs {
     type Output = ();
 
     fn run(self) -> eyre::Result<Self::Output> {
-        let config: Config = From::from(&self);
+        let config = self.load_config_emit_warnings();
```

Comment:
nice, I like this solution better than having to do `let config: Config = From::from(&self);`